fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -457,6 +457,11 @@ public class SessionVariable implements Serializable, Writable {
             SHOW_HIDDEN_COLUMNS
     );
 
+    public static final String SHUFFLE_FACTOR_LOWER_BOUND = "shuffle_factor_lower_bound";
```

Comment:
remove this by using a fixed value (we have hint control as a no-brainer plan) or add upper side also